test/models/quote_test.exs

```diff
@@ -11,6 +15,16 @@ defmodule HonGraffitiPhoenix.QuoteTest do
     assert changeset.valid?
   end
 
+  test "validate requires at least a minumum length" do
+    changeset = Quote.changeset(%Quote{}, @short_attrs)
```

Comment:
I'm a fan of the [four-phase testing pattern](https://robots.thoughtbot.com/four-phase-test), by which I mostly mean grouping those four phases together (or however many are in your test), separating each group by a newline. It's trivial in the case of these tests as they're so tiny, but I think a useful way to organize the contents of tests, in general.